src/verifiers/documentIntegrity/hash/openAttestationHash.test.ts

```diff
@@ -134,7 +134,7 @@ describe("OpenAttestationHash", () => {
         },
       ]);
     });
-    it("should fail if the merkle root is missing", async () => {
+    it("should skip if the merkle root is missing", async () => {
```

Comment:
Here, we don't run the verification method anymore, because the file is not considered valid.